FoxHound.Client/src/Content.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  useRouteMatch,
+  useParams,
```

Comment:
Remove all imports other than Switch and Route since the others aren't being used by this component